pkg/clusteragent/admission/mutate/config.go

```diff
@@ -118,6 +142,23 @@ func injectConfig(pod *corev1.Pod, _ string, _ dynamic.Interface) error {
 
 	injectedEntity = injectEnv(pod, ddEntityIDEnvVar)
 
+	// Inject env variables used for Onboarding KPIs propagation
+	if isApmInstrumentationEnabled(pod.Namespace) {
+		// if Single Step Instrumentation is enabled, inject DD_INSTRUMENTATION_INSTALL_TYPE:k8s_single_step
+		_ = injectEnv(pod, singleStepInstrumentationInstallTypeEnvVar)
+	} else {
+		// if local library injection is enabled, inject DD_INSTRUMENTATION_INSTALL_TYPE:k8s_lib_injection
+		_ = injectEnv(pod, localLibraryInstrumentationInstallTypeEnvVar)
+	}
```

Comment:
I think we are missing an install type, when the user already have their application instrumented with APM and they just need to inject the configuration.
 
In that cas `isApmInstrumentationEnabled` might return false, but it is not an lib injection nether...

Reply:
That's a good point. In this case we do need to check pod annotations - if library annotation is specified, then it's the case of local library injection, otherwise it's manual instrumentation. Does it make sense?

Reply:
> I think we are missing an install type, when the user already have their application instrumented with APM and they just need to inject the configuration.

This is expected. In this case, the trace agent will apply a "manual" instrumentation type when none is provided previously.

Reply:
but this PR change the code of an admission controller that can be use in standalone. 
how the trace-agent will understand it?

Reply:
So how this works is that the Cluster Agent injects these environment variables in which the APM libraries will read and attach to traces. The Trace Agent will read the install_time and compute the time-to-first-trace and tag it by the install type.

Reply:
I think I know this part.
but how the trace-agent make a different between:
* library added by the user in their container image
* library added by the admission controller?